symbols/overlay29.yml

```diff
@@ -1389,6 +1389,29 @@ overlay29:
         Zeroes the damage data struct, which is output by the damage calculation function.
         
         r0: damage data pointer
+    - name: FreeLoadedAttackSpriteAndMore
+      address:
+        EU: 0x22F78C8
+      description: Free among other another data structure the attack sprite storage area/data
```

Comment:
When a function has no inputs or outputs, convention is to put `No params." where the parameter list would normally go. In contrast, not saying anything is used to mean that we don't know the inputs or outputs.